admin-dev/themes/new-theme/js/components/pagination/dynamic-paginator.ts

```diff
@@ -178,13 +150,43 @@ export default class DynamicPaginator {
     window.prestaShopUiKit.initToolTips();
   }
 
+  /**
+   * Initiates the pagination component
+   *
+   * @private
+   */
+  private init(): void {
+    this.$paginationContainer.on('click', this.selectorsMap.pageLink, (e) => {
+      this.paginate(Number($(e.currentTarget).data('page')));
+    });
+    this.$paginationContainer
+      .find(this.selectorsMap.jumpToPageInput)
+      .keypress((e) => {
+        if (e.which === 13) {
+          e.preventDefault();
+          const input = <HTMLInputElement>e.currentTarget;
+          const page = this.getValidPageNumber(Number(input.value));
+          this.paginate(page);
+        }
+      });
+    this.$paginationContainer.on(
```

Comment:
you can chain the 2 binds (`on`) but it's a detail

Reply:
I agree but the code was simply moved, so it's ok